src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/cas2/TimelineEventsTransformer.kt

```diff
@@ -25,7 +25,7 @@ class TimelineEventsTransformer {
         type = TimelineEventType.cas2Note,
         occurredAt = it.createdAt.toInstant(),
         label = "Note",
-        createdByName = it.createdByNomisUser.name,
+        createdByName = it.getCreatedByNomisOrExternalUser().name,
```

Comment:
I wonder if the timeline transformer needs to know about the underlying implementation? Would `getUser` suffice?

Reply:
oh that was about a different method... yeah I think `getUser` would be better here.